junit-plugin/src/main/scala/org/scalajs/junit/plugin/ScalaJSJUnitPlugin.scala

```diff
@@ -0,0 +1,447 @@
+package org.scalajs.junit.plugin
+
+import scala.language.reflectiveCalls
+
+import org.scalajs.junit._
+
+import scala.reflect.internal.Flags
+import scala.tools.nsc._
+import scala.tools.nsc.plugins.{
+Plugin => NscPlugin, PluginComponent => NscPluginComponent
+}
+
+class ScalaJSJUnitPlugin(val global: Global) extends NscPlugin {
+  selfPlugin =>
+
+  val name: String = "Scala.js JUnit plugin"
+
+  val components: List[NscPluginComponent] =
+    List(ScalaJSJUnitPluginComponent)
+
+  val description: String = "Adapts JUnit test classes to Scala.js"
```

Comment:
How about "Makes JUnit test classes invokable in Scala.js"?

Mere adaptation seems strange, since it is (IIUC) not necessary.